lib/surveyor/models/response_set_methods.rb

```diff
@@ -9,12 +9,18 @@ module ResponseSetMethods
 
       included do
         # Associations
-        belongs_to :survey
-        belongs_to :user, required: false
-        belongs_to :current_section, foreign_key: :current_section_id, class_name: :survey_section, required: false
+        belongs_to :survey, optional: false
```

Comment:
`optional: false` is the default -- we don't need to add it here.

Reply:
FYI: I was leaving it here so the gem would not be dependent on someone's rails config

Reply:
Makes sense.